layers/+tags/gtags/README.org

```diff
@@ -5,121 +5,249 @@
-- Jump to a location based on context
-- Find definitions
-- Find references
-- Present tags in current function only
-- Create a tag database
-- Jump to definitions in file
-- Show stack of visited locations
-- Manually update tag database
-- Jump to next location in context stack
-- Jump to previous location in context stack
-- Jump to a file in tag database
+  - Select any tag in a project retrieves by gtags
+  - Resume previous helm-gtags session
+  - Jump to a location based on context
+  - Find definitions
+  - Find references
+  - Present tags in current function only
+  - Create a tag database
+  - Jump to definitions in file
+  - Show stack of visited locations
+  - Manually update tag database
+  - Jump to next location in context stack
+  - Jump to previous location in context stack
+  - Jump to a file in tag database
+  - Enables =eldoc= in modes that otherwise might not support it.
+  - Enables =company complete= in modes that otherwise might not support it.
 
 * Install
-** GNU Global
 
-You can install =helm-gtags= from the software repository of your OS. For example, in Ubuntu:
+** GNU Global (gtags)
 
-#+BEGIN_SRC sh
-  sudo apt-get install global
-#+END_SRC
+   To use =gtags=, you first have to install [[https://www.gnu.org/software/global/download.html][GNU Global]].
 
-To use =helm-gtags=, you first have to install [[https://www.gnu.org/software/global/download.html][GNU Global]].
+   You can install =global= from the software repository of your OS; however,
+   many OS distributions are out of date, and you will probably be missing
+   support for =pygments= and =exuberant ctags=, and thus support for many
+   languages. We recommend installing from source. If not for example to install
+   on Ubuntu:
 
-Download the latest tar.gz archive, then run these commands:
+   #+begin_src sh
+     sudo apt-get install global
+   #+end_src
 
-#+BEGIN_SRC sh
-  tar xvf global-6.4.tar.gz
-  cd global-6.4
-  ./configure
-  make
-  sudo make install
-#+END_SRC
+*** Install on OSX using Homebrew
 
-To be able to use =ctags= and other backends, you need to copy the sample
-=gtags.conf= either to =/etc/gtags.conf= or =$HOME/.globalrc=. For example:
+    #+begin_src sh options
+      brew install global --with-pygments --with-ctags
+    #+end_src
 
-#+begin_src sh
-  cp gtags.conf ~/.globalrc
-#+end_src
+*** Install on *nix from source
 
-To use this configuration layer, add it to your =~/.spacemacs=. You will need to
-add =gtags= to the existing =dotspacemacs-configuration-layers= list in this
-file.
+**** Install recommended dependencies
 
-* Usage
+    To take full advantage of global you should install 2 extra packages in
+    addition to global: pygments and ctags (exuberant). You can do this using
+    your normal OS package manager, e.g., on Ubuntu
+
+    #+BEGIN_SRC sh
+      sudo apt-get install exuberant-ctags python-pygments
+    #+END_SRC
+
+    or e.g., Archlinux:
+
+    #+BEGIN_SRC sh
+      sudo pacman -S ctags python-pygments
```

Comment:
Joe Hillenbrand <notifications@github.com> writes:
...
> global is also available in the
> [AUR](https://aur.archlinux.org/packages/global/). Is that needed as
> well?

Well the text is describing installing the requirements before
compiling global form source. I thought about mentioning AUR as well,
but since there's no standard command for installing from AUR it's not
really "provided by the OS package system" like yum or apt-get is. Plus,
most everyone running archlinux probably knows to check AUR anyway :)

Thanks,
Chris.